### PR TITLE
Add waiter for the testing addons

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Add waiter for the testing addons

**Related issue(s)**

### CODE REVIEW

File: installation/scripts/testing-common.sh [hunk 1]
```diff
@@ -217,6 +217,18 @@ spec:
...
+        msg=$(kubectl get clusteraddonsconfiguration ${TESTING_ADDONS_CFG_NAME} -o=jsonpath='{.status.phase}')
+        if [[ ${msg} = "Ready" ]]; then
+            break
+        else
+            echo "waiting for ready testing addons..."
+            sleep 2
+        fi
+        retry=$[retry + 1]
+    done
+
```
Comment (mszostok) [inline installation/scripts/testing-common.sh]: and what if status will not become Ready???

Comment (polskikiel) [reply installation/scripts/testing-common.sh]: added error handling

File: installation/scripts/testing-common.sh [hunk 2]
```diff
@@ -217,7 +217,20 @@ spec:
   repositories:
   - url: "https://github.com/kyma-project/addons/releases/download/0.7.0/index-testing.yaml"
 EOF
-    log "Testing addons injected" green
+    local retry=0
+    while [[ ${retry} -lt 10 ]]; do
+        msg=$(kubectl get clusteraddonsconfiguration ${TESTING_ADDONS_CFG_NAME} -o=jsonpath='{.status.phase}')
+        if [[ ${msg} = "Ready" ]]; then
```
Comment (mszostok) [inline installation/scripts/testing-common.sh]:
```suggestion
        if [[ "${msg}" = "Ready" ]]; then
```

File: installation/scripts/testing-common.sh [hunk 3]
```diff
@@ -217,7 +217,20 @@ spec:
...
+            log "Testing addons injected" green
+            return 0
+        else
+            echo "waiting for ready testing addons..."
```
Comment (mszostok) [inline installation/scripts/testing-common.sh]: info about retry cnt

**Pull request merged**